src/server/templates/estate/EstateDefault.jsx

```diff
@@ -5,12 +5,9 @@ import {MainLayout} from '../../layouts';
 export const EstateDefault = () => {
     const {currentNode} = useServerContext();
     return (
-        <MainLayout
-            head={<meta name="description" content="content template"/>}
```

Comment:
@herve Removing as it conflicts with the new SEO fields. just want to check if you have thoughts on it (as it seems you've added it in), or if ok for now.